flake.nix

```diff
@@ -0,0 +1,63 @@
+{
+  description = "EvaP";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    poetry2nix = {
+      url = "github:nix-community/poetry2nix";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+
+    flake-parts.url = "github:hercules-ci/flake-parts";
```

Comment:
Just saw https://github.com/Platonic-Systems/process-compose-flake/pull/80, I think I will check it out to see if we can get rid of this dependency then